### ISSUE TITLE
Add the ability to Submit multiple unreported expenses

### ISSUE THREAD

**Issue by trjExpensify** [Expensify/App#53062]
Coming from [here](https://expensify.slack.com/archives/C07HPDRELLD/p1731696382976459). 

Just creating this as a placeholder for now, assigned to myself. There's a few moving parts for this one that we need or can potentially impact the design, so not jumping the gun. 

- Report-first flow(s)
- Adding action buttons to the search page 
- Suggested searches, so we can use one for "Unreported expenses"

**Comment by trjExpensify** [comment]
No change for now, still holding this.

**Comment by trjExpensify** [comment]
Same, progressing the wider plan.

**Comment by trjExpensify** [comment]
Same as above, Melv.

**Comment by trjExpensify** [comment]
Same as above, Melv. Going to actually drop this to monthly for now.

**Comment by trjExpensify** [comment]
Same.

**Comment by danielrvidal** [comment]
I'm guessing this is still not a priority.

**Comment by trjExpensify** [comment]
Right, in a chain of dependencies for related functionality to support it. Creating reports, moving expenses between reports etc. After those are in, we can re-evaluate if there's something to do here.

**Comment by danielrvidal** [comment]
Just noting that we're still holding.

**Comment by trjExpensify** [comment]
Same, progressing TradReports.

**Comment by trjExpensify** [comment]
I actually think @dylanexpensify project is going to take care of this, so I'm not sure how much this issue is necessary. 

What we have today: 

- You can move unreported expenses to an existing open/processing report 
- You can add unreported expenses to an empty report in the report creation flow. 
- You can filter on the Reports page for `status:unreported` expenses. 

What we don't have:
- Create a new report by bulk selecting multiple unreported expenses
- Code unreported expenses using your default coding. 

... but those items are coming with the unreported improvements project of Dylan's. With that, I'm cool to close this out. @garrettmknight whatcha' think?
